subprojects/parseq/src/test/java/com/linkedin/parseq/trace/TestShallowTraceBuilder.java

```diff
@@ -97,27 +104,56 @@ public void testEquals() {
     builderCopy.setName("no-test");
     builderCopy.setResultType(ResultType.SUCCESS);
     builderCopy.setValue("value");
-    builderCopy.setStartNanos(123L);
-    builderCopy.setPendingNanos(234L);
-    builderCopy.setEndNanos(345L);
+    builderCopy.setNativeStartNanos(123L);
+    builderCopy.setNativePendingNanos(234L);
+    builderCopy.setNativeEndNanos(345L);
     assertFalse(builder.build().equals(builderCopy.build()));
 
     builderCopy = new ShallowTraceBuilder(100L);
     builderCopy.setName("test");
     builderCopy.setResultType(ResultType.SUCCESS);
-    builderCopy.setStartNanos(123L);
-    builderCopy.setPendingNanos(234L);
-    builderCopy.setEndNanos(345L);
+    builderCopy.setNativeStartNanos(123L);
+    builderCopy.setNativePendingNanos(234L);
+    builderCopy.setNativeEndNanos(345L);
     assertFalse(builder.build().equals(builderCopy.build()));
 
     builderCopy = new ShallowTraceBuilder(100L);
     builderCopy.setName("no-test");
     builderCopy.setResultType(ResultType.SUCCESS);
     builderCopy.setValue("value");
-    builderCopy.setStartNanos(123L);
-    builderCopy.setPendingNanos(234L);
-    builderCopy.setEndNanos(345L);
+    builderCopy.setNativeStartNanos(123L);
+    builderCopy.setNativePendingNanos(234L);
+    builderCopy.setNativeEndNanos(345L);
     builderCopy.setResultType(ResultType.ERROR);
     assertFalse(builder.build().equals(builderCopy.build()));
   }
+
+  @Test
+  public void testDeprecatedTimeMethods() {
```

Comment:
Let's cover several permutations for those fields we touched.
1. set and get all using deprecated methods
2. set using new method, and get using deprecated method
3. set using deprecated method, and get using new method
4. set and get all using new methods (which should already covered by all other tests you have fixed)

Reply:
updated